docs/infrastructure/README.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+Student Robotics has a variety of infrastrcture in order to support its mission.
+This document aims to provide an outline of the general structure.
+
+Currently our infrastructure as a whole does not have a designed owner, instead
+individual pieces are looked after by volunteer Teams or just by volunteers who
+have historically maintained them.
+
+## Website
+
+Our website consists of a main public facing site at https://studentrobotics.org
+as well as a collection of other sites which appear at various paths off the
+root domain. This runbook is an example of such a sub-site, as are the
+competitor-facing documentation which appears at [`/docs`][slash-docs].
+
+These sub sites (and the main website itself) are hosted as GitHub Pages, with a
+separate NGINX instance acting as a [reverse-proxy][reverse-proxy] to facade
+these and other services onto the root domain. Currently we use Digital Ocean
+for the hosting of our [infrastructure][infrastructure] as well as for DNS and
+TLS termination. Our domains are registered with Namecheap.
```

Comment:
```suggestion
TLS termination. Our domains are registered with Namecheap and are managed by the Trustees.
```

Reply:
This is not strictly true -- the DNS entries for them are in Digital Ocean (as this paragraph notes) and the actual technical management of the domains is therefore a bit grey in terms of who manages it. I've deliberately opted to leave that detail out for now. Once the Infrastructure Team exists, this will naturally become clearer (and an easy fix to document).